redwood-treehouse-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/treehouse/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -36,13 +36,19 @@ import platform.UIKit.row
 import platform.UIKit.section
 import platform.darwin.NSInteger
 
-internal class UIViewLazyColumn<A : AppService>(
+internal class UIViewLazyList<A : AppService>(
   treehouseApp: TreehouseApp<A>,
   widgetSystem: WidgetSystem,
-) : LazyColumn<UIView> {
+) : LazyList<UIView> {
   private val dataSource = TableViewDataSource(treehouseApp, widgetSystem)
   private val root = UITableView().apply {
-    this.dataSource = this@UIViewLazyColumn.dataSource
+    this.dataSource = this@UIViewLazyList.dataSource
+  }
+
+  override fun isVertical(isVertical: Boolean) {
+    if (!isVertical) {
+      TODO()
```

Comment:
I wonder if the TODO action catches `TODO()`. Maybe put a proper `// TODO` to make sure it gets tracked?